internal/service/cloudbackupschedule/model_cloud_backup_schedule_test.go

```diff
@@ -0,0 +1,164 @@
+package cloudbackupschedule_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/service/cloudbackupschedule"
+	"go.mongodb.org/atlas-sdk/v20240530002/admin"
+)
+
+func TestFlattenPolicyItem(t *testing.T) {
+	testCases := []struct {
+		name          string
```

Comment:
nit: We have been going for the approach of defining unit test cases as a map with the key being the name ([example](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/service/advancedcluster/model_advanced_cluster_test.go#L82))